git_automation.py

```
@@ -0,0 +1,64 @@
+        return False
+
+def main():
+    os.chdir('/home/vibecoder/neonpro')
+    
+    # Step 1: Add all changes
+    if not run_git_command('git add .', 'Adding all changes to staging'):
+        sys.exit(1)
+    
+    # Step 2: Check if there are changes to commit
+    try:
+        result = subprocess.run('git diff --cached --quiet', shell=True, cwd='/home/vibecoder/neonpro')
```

Ruff (0.12.2)

`subprocess` call with `shell=True` seems safe, but may be changed in the future; consider rewriting without `shell`

(S602)